<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPlayBackOptionsPanel</class>
 <widget class="QWidget" name="DPlayBackOptionsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DPlayBackOptionsPanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Color</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="styleText_label">
        <property name="text">
         <string> Style:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="style_comboBox">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cellLabels_groupBox">
     <property name="title">
      <string>Labels</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="cellLabelText_label">
        <property name="text">
         <string> Shows:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cellLabel_comboBox">
        <item>
         <property name="text">
          <string>No Text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cell Occupancy Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cell Occupancy Percentage%</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>244</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_pushButton">
       <property name="text">
        <string>Exit Playback</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
